### ISSUE TITLE
'my X' as address/endearment

### ISSUE THREAD

**Issue by esmanning** [carmls/snacs-guidelines#98]
Addressing someone with 'MY [noun]', where the noun doesn't explicitly express a social relationship, but the construction seems to indicate familiarity/endearment?

e.g. in Little Prince ch 6:

> But on your tiny planet, MY little prince, all you need to do is move your chair a few steps.
sent_id = lpp_1943.282

Could also be seen in more everyday situations with e.g. 'my dude'

Is this SocialRel~Gestalt? Or just Gestalt?

**Comment by nschneid** [comment]
It has to be "my", right? Not "your"/"their"?

Feels like a special discursive construction. Maybe \`d or \`$?

**Comment by esmanning** [comment]
Yeah, would have to be "my" for this construction. `$ feels like a better fit to me than `d.

**Comment by nschneid** [comment]
`$ sounds good
